tardis/transport/montecarlo/tests/test_cuda_formal_integral.py

```diff
@@ -122,31 +119,26 @@ def formal_integral_geometry(request):
 
 
 @pytest.fixture(scope="function")
-def formal_integral_model():
+def time_explosion():
     """
-    This gets the Numba model to be used in later tests
+    This gets the time_explosion to be used in later tests
     """
-    model = NumbaModel(
-        1 / c.c.cgs.value,
-    )
-    return model
+    return 1 / c.c.cgs.value
```

Comment:
what is that?

Reply:
maybe make this a well named constasnt.

Reply:
`C_INV` is used in formal integral so I'll grab that